src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -32,6 +32,8 @@ private EmptyDeadline() {
 
     private LocalDate projectDeadline;
 
+    private String deadlineString;
```

Comment:
Not sure why additional deadline string is stored when it can be fetched from the projectDeadline object - but not a major thing

Reply:
Change made